packages/react-components/react-infobutton/src/components/InfoTip/useInfoTip.ts

```diff
@@ -12,17 +14,40 @@ import type { InfoTipProps, InfoTipState } from './InfoTip.types';
  * @param ref - reference to root HTMLElement of InfoTip
  */
 export const useInfoTip_unstable = (props: InfoTipProps, ref: React.Ref<HTMLElement>): InfoTipState => {
+  const { size = 'medium', info } = props;
+  const [open, setOpen] = React.useState(false);
+
+  const tooltip = resolveShorthand(props.tooltip, {
+    required: true,
+    defaultProps: {
+      content: info,
+      positioning: 'above-start',
+      relationship: 'description',
+      withArrow: true,
+    },
+  });
+
+  tooltip.onVisibleChange = mergeCallbacks(tooltip.onVisibleChange, (e, data) => {
+    setOpen(data.visible);
+  });
+
   return {
-    // TODO add appropriate props/defaults
+    open,
+    size,
+
     components: {
-      // TODO add each slot's element type or component
-      root: 'div',
+      root: 'span',
+      tooltip: Tooltip as React.FC<Partial<TooltipProps>>,
     },
-    // TODO add appropriate slots, for example:
-    // mySlot: resolveShorthand(props.mySlot),
-    root: getNativeElementProps('div', {
-      ref,
+
+    root: getNativeElementProps('span', {
+      'aria-label': 'information',
```

Comment:
I think I'd lean towards not including this by default, and having docs suggesting adding a short one-word `aria-label` with something similar to "information".

The reason being that I think the best word to use is a bit contextual (even aside from localization), and including an extra word before the tooltip text is helpful context but not actually necessary